lib/utils/index.js

```diff
@@ -0,0 +1,7 @@
+function getCrossorigin (crossorigin, defaultCrossOrigin) {
+  crossorigin = crossorigin === true ? 'anonymous' : crossorigin;
```

Comment:
without true as value this function has to be refactored

Reply:
Why without `true`? 
Behavior of `crossorigin` would be `anonymous` with `true`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin